### ISSUE TITLE
Cannot call methods

### ISSUE THREAD

**Issue by draft6** [fullstorydev/grpcurl#73]
I have a very simple test setup with server reflection enabled - no proxies or multiple servers, just a single service exposing a few methods.

I can successfully list methods with: 
```
grpcurl -plaintext localhost:50051 describe test.MyService.Login
test.MyService.Login is a method:
rpc Login ( .test.LoginRequest ) returns ( .test.LoginReply ) {
  option (.google.api.http) = { post:"/login" };
}
```

As well as get info on the types:
```
grpcurl -msg-template -plaintext localhost:50051 describe test.LoginRequest
test.LoginRequest is a message:
message LoginRequest {
  string email = 1;
  string password = 2;
}

Message template:
{
  "email": "",
  "password": ""
}
```

But, if i try to call a method - it fails: 
```
grpcurl -plaintext -d '{"email":"test", "password":"test"}' localhost:50051 test.MyService/Login
Failed to construct request parser and formatter for "json": error creating message resolver: Symbol not found: grpc.reflection.v1alpha.ServerReflection

```
Any ideas? 
Worth mentioning is that Polyglot works with no issues.

**Comment by draft6** [comment]
I would say the problem is related to the { option (.google.api.http) = { post:"/login" };} part, an unresolved symbol?

**Comment by jhump** [comment]
@draft6, could you pull down latest code for this repo, checkout `origin/jh/fix-potential-refl-issue`, and then see if that fixes your issue or not? I think it should, but I'd love to know if you could try it out.

Also, out of curiosity, what language/runtime is the server in question?

**Comment by draft6** [comment]
@jhump The server is **C++** and yes, this branch fixes the issue, thank you!

I will leave the issue open so you can close it on your convenience once the fixes are in place.

**Comment by jhump** [comment]
Fixed by #74
